src/main/kotlin/com/playwright/remote/engine/route/UrlMatcher.kt

```diff
@@ -8,7 +8,7 @@ import java.util.regex.Pattern.compile
 
 typealias Predicate = (String) -> Boolean
 
-class UrlMatcher(private val rawSource: Any = "", private val predicate: Predicate = { false }) {
+class UrlMatcher(private val rawSource: Any = "", private val predicate: Predicate? = { false }) {
```

Comment:
I think if you make Predicate nullable then its default value should be null.

Reply:
Fixed